livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -865,6 +851,29 @@ async def set_chat_ctx(self, new_ctx: llm.ChatContext) -> None:
             },
         )
 
+        # append an empty audio message if all messages are text
+        if new_ctx.messages and all(
+            isinstance(msg.content, str) for msg in new_ctx.messages
+        ):
```

Comment:
Can you check if `audio` is inside the modality

Reply:
Changed to 
```
if new_ctx.messages and not any(
            isinstance(msg.content, llm.ChatAudio) for msg in new_ctx.messages
        ):
```